data/spawns/npc-spawns.yml

```diff
@@ -1,5 +1,7 @@
 # 6743
 - { id: banker_fist_of_guthix, x: 1705, y: 5599 }
+# 6993
+- { id: giant_mole, x: 1762, y: 5185 }
```

Comment:
Would be nice to have some of the baby moles walking about 😄

Reply:
I did think the mole looked abit lonely...